### PR TITLE
A dockerfile is created to include nodejs in a single container.

### PR DESCRIPTION
**Pull request by uetiko**
## Problem

have two containers and two docker-compose file to run the project.

## Solution

A dockerfile is created that extends the indava image to which nodejs is included and thus  eliminates the second docker-compose and having all tools in a single container to keep the project.

## Test

run docker compose and enter in the container for run composer and yarn.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,29 @@
...
 ```sh
 docker-compose up -d
 docker-compose exec -u web web bash
-./bin/console doctrine:migra:migra
+```
+
+- Install dependencies
+```bash
+composer install -o
+yarn install
```
Comment (jashk) [inline README.md:22]: This command is already in `Build assets` section

```suggestion
```

File: dockerfiles/php/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from node:12.18.1-slim AS node
+
+from indava/php:7.4
```
Comment (jashk) [inline dockerfiles/php/Dockerfile:3]: I think we need to keep PHP 7.3 until have enough test to migrate to PHP 7.4

File: README.md (hunk 1 above)
Comment (uetiko) [reply README.md:30]: Thank you very much. The README has been modified to clarify this. Now it is detailed how to install the dependencies and how to do the build separately.

Comment (uetiko) [reply README.md:30]: Thank you very much. The README has been modified to clarify this. Now it is detailed how to install the dependencies and how to do the build separately.

File: dockerfiles/php/Dockerfile (hunk 2 above)
Comment (uetiko) [reply dockerfiles/php/Dockerfile]: Fixed

File: README.md [hunk 3]
```diff
@@ -6,23 +6,50 @@
...
 docker-compose up -d
 docker-compose exec -u web web bash
-./bin/console doctrine:migra:migra
 ```
 
-- Build assets
-```sh
-docker-compose -f docker-compose.cli.yml run --rm yarn install
-docker-compose -f docker-compose.cli.yml run --rm yarn build
+To install the necessary dependenceies of the project, both for php
```
Comment (jashk) [inline README.md:25]:
```suggestion
To install the necessary dependencies of the project, both for php
```
Typo

File: README.md [hunk 4]
```diff
@@ -6,23 +6,50 @@
...
+and javascript, inside the container execute the following commands.
```
Comment (jashk) [inline README.md:26]:
```suggestion
and javascript, execute the following commands inside the container.
```
I think this is more accurate, what do you think?

File: README.md [hunk 5]
```diff
@@ -6,23 +6,50 @@
...
+For build assets, you only need run the following command, remember
+run this command every time when you add changes to styles files or
+javascript files.
+
+```sh
+yarn install
+yarn build
+```
 
+To see the portal of phpmx, write the address `http://localhost:8080/`
```
Comment (jashk) [inline README.md:50]:
```suggestion
To see the portal of phpmx, write the address [http://localhost:8080/](http://localhost:8080/) 
```

File: dockerfiles/php/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,5 @@
+from node:12.18.1-slim AS node
```
Comment (jashk) [inline dockerfiles/php/Dockerfile:1]:
```suggestion
FROM node:12.18.1-slim AS node
```

File: dockerfiles/php/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+from indava/php:7.3
```
Comment (jashk) [inline dockerfiles/php/Dockerfile:3]:
```suggestion
FROM indava/php:7.3
```

File: README.md (hunk 5 above)
Comment (uetiko) [reply README.md:50]: Being a free general purpose manual for anyone who can read it in a repository, putting a link that is only active in local mode and as long as the project is executed, I do not see the need for this change in the documentation.

**Comment by jashk** [comment]
Hi @dmouse, please check if we can merge this PR

**Pull request closed without merging**